recipes-bsp/optee-os/optee-os-iot2050_3.18.0.bb

```diff
@@ -21,20 +21,17 @@ OPTEE_NAME = "iot2050"
 
```

Comment:
3.19.0 released, which carries a patch that we have interest into:

https://github.com/OP-TEE/optee_os/commit/115198b44dffbe088b4aec0ff82a7d32d7d704d2

Reply:
OK, will update.